one_fm/patches/v15_0/update_job_offer_00057_erf.py

```diff
@@ -0,0 +1,10 @@
+import frappe
```

Comment:
please, Did you check out from version-15 ?

Reply:
I didn't touch this either